astarte/device/mapping.py

```diff
@@ -167,6 +187,9 @@ def validate(self, payload: MapType, timestamp: datetime) -> ValidationError | N
         """
         min_supported_int = -2147483648
         max_supported_int = 2147483647
+        # Make sure the payload contains at least something
```

Comment:
maybe something like this is better
```suggestion
        # Make sure the payload is not empty
```
 but is this comment helpful? I think the description of Validation Error is already comprehensive, so what do you think to remove the comment?